test/specs/behaviors/testHelper.tsx

```diff
@@ -60,25 +67,39 @@ export class TestHelper {
   public findRegexAndAssingCorrespondingInfoToArray(behaviorMenuItems: any) {
```

Comment:
why `behaviorMenuItems`?

Reply:
yes this is from historical reasons, we named it like it...
and this is basically whole object of loaded file "behaviorMenu.json"